connector-webhook/src/main/java/com/redhat/cloud/notifications/connector/webhook/WebhookRouteBuilder.java

```diff
@@ -0,0 +1,75 @@
+package com.redhat.cloud.notifications.connector.webhook;
+
+import com.redhat.cloud.notifications.connector.ConnectorConfig;
+import com.redhat.cloud.notifications.connector.EngineToConnectorRouteBuilder;
+import com.redhat.cloud.notifications.connector.webhook.authentication.BasicAuthenticationProcessor;
+import com.redhat.cloud.notifications.connector.webhook.authentication.BearerTokenAuthenticationProcessor;
+import com.redhat.cloud.notifications.connector.webhook.authentication.InsightsTokenAuthenticationProcessor;
+import org.apache.camel.builder.endpoint.dsl.HttpEndpointBuilderFactory;
+import org.apache.camel.component.http.HttpComponent;
+import org.apache.http.conn.ssl.NoopHostnameVerifier;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import static com.redhat.cloud.notifications.connector.ConnectorToEngineRouteBuilder.SUCCESS;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.*;
+import static com.redhat.cloud.notifications.connector.webhook.ExchangeProperty.BASIC_AUTH_USERNAME;
+import static com.redhat.cloud.notifications.connector.webhook.ExchangeProperty.BEARER_TOKEN;
+import static com.redhat.cloud.notifications.connector.webhook.ExchangeProperty.INSIGHT_TOKEN_HEADER;
+import static com.redhat.cloud.notifications.connector.webhook.ExchangeProperty.TARGET_URL_NO_SCHEME;
+import static com.redhat.cloud.notifications.connector.webhook.ExchangeProperty.TRUST_ALL;
+import static com.redhat.cloud.notifications.connector.webhook.SslTrustAllManager.getSslContextParameters;
+import static org.apache.camel.LoggingLevel.INFO;
+
+
+@ApplicationScoped
+public class WebhookRouteBuilder extends EngineToConnectorRouteBuilder {
+
+    @Inject
+    ConnectorConfig webhookConnectorConfig;
+
+    @Override
+    public void configureRoute() {
```

Comment:
This method needs to deal with both `http` and `https` URLs since we're allowing them in the webhook properties.

FYI, the timeouts config won't work with `toD` so it would be better if we always constructed the Camel endpoint using `getCamelContext().getComponent("http", HttpComponent.class)` or `getCamelContext().getComponent("https", HttpComponent.class)` depending on the URL scheme.

`TRUST_ALL` should only be used when the scheme is `https`.

Reply:
After few tests, it appears that timeout configuration also works with to `toD`. As you recommand, I also applied it to `http` calls.